src/server/game/Chat/LanguageMgr.cpp

```diff
@@ -251,7 +247,7 @@ std::string LanguageMgr::Translate(std::string const& msg, uint32 language, Loca
                         size_t length = std::min(wstrSourceWord.length(), strlen(replacementWord));
                         for (size_t i = 0; i < length; ++i)
                         {
-                            if (isUpper(wstrSourceWord[i]))
+                            if (wstrSourceWord[i] != L'\'' && isUpper(wstrSourceWord[i]))
```

Comment:
This is redundant here, neither ToUpper nor ToLower transform this character

Reply:
transformation is applied to replacementWord[i] not char '